docs/html/topics/dependency-resolution.md

```diff
@@ -0,0 +1,107 @@
+# Dependency Resolution
+
+pip is capable of determining and installing the dependencies of packages. The
+process of determining which version of a dependency to install is known as
+dependency resolution. This behaviour can be disabled by passing
+{any}`--no-deps` to {any}`pip install`.
```

Comment:
I wonder if we should encourage the full form `--no-dependencies` here.

Reply:
🤷🏽 

I feel like nearly everyone types `deps` rather than `dependencies` simply because it's shorter and equally clear in context.